projects/dark-matter-reana.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: REANA workflow for Dark Matter Searches
+postdate: 2023-02-14
+categories:
+  - Analysis tools
+durations:
+  - 3 months
+skillset:
+  - Python
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - SuperCDMS
+program:
+  - Any
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Implement a REANA workflow for dark matter searches at SuperCMDS
+description: >
+  [REANA](https://reana.io/) is a platform for reproducible data analysis workflows
+  that can be run at scale. REANA has been used extensively for running containerized
+  workflows of LHC experiments, like ATLAS, and for reinterpretation of published
+  analyses. This project would aim to implement a REANA workflow for a dark matter
+  search at SuperCMDS to improve replicability of analyses and to provide a starting
+  point for future SuperCDMS analyses.
+contacts:
+  - name: Matthew Feickert
+    email: matthew.feickert@cern.ch
+  - name: Lukas Heinrich
+    email: lukas.heinrich@cern.ch
+  - name: Amy Roberts
+    email: amy.roberts@ucdenver.edu
```

Comment:
I put @lukasheinrich and myself down for contacts as we discussed this in the IRIS-HEP Slack. I added @pibion as a contact given the discussions had with @cranmer on the idea, but I'm not sure if you have the time to be a mentor or not. @kratsg if you'd like to be a mentor on this as well please just add yourself in a comment and I'll add it in.